frontend/amundsen_application/static/js/interfaces/User.ts

```diff
@@ -6,6 +6,7 @@ export interface User {
   email: string;
   profile_url: string;
   user_id: string;
+  other_key_values?: Record<string, string>;
```

Comment:
what is this? I think I vaguely remember you did something with additional values for users, but can't remember the details. thinking this name should be made a little more descriptive so people know what it is/can be used for

Reply:
It was named `other_key_values` on the backend User model so I thought the pattern might be just match that naming here?